src/main/java/org/folio/rmapi/model/CustomerResources.java

```diff
@@ -0,0 +1,160 @@
+package org.folio.rmapi.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.List;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class CustomerResources {
```

Comment:
I am a little curious about this model - is this related to this documentation here? https://s3.amazonaws.com/foliodocs/api/mod-kb-ebsco/eholdings.html#eholdings_resources__resourceid__get

Reply:
This model corresponds to response from RM API, 
I've taken it from here 
https://developer.ebsco.com/interactive/holdingsiq#/